Only limit key types for `ops` keys (#710)
decentralized-identity/sidetree: docs/spec/patches.md

2. The object ****MUST**** include a `public_keys` property, and its value ****MUST**** be an array.
3. Each key being added ****MUST**** be represented by an entry in the `public_keys` array, and each entry must be an object composed as follows:
    1. The object ****MUST**** include an `id` property, and its value ****MUST**** be a string with no more than twenty (20) ASCII encoded characters. If the value is not of the correct type or exceeds the specified length, the entire _Patch Action_ ****MUST**** be discarded, without any of it being used to modify the DID's state.

>     2. The object ****MUST**** include a `type` property, and its value ****SHOULD**** be the identifier type string of a registered [Cryptographic Suite](https://w3c-ccg.github.io/ld-cryptosuite-registry/) that supports the `publicKeyJwk` representation (examples below). If the key usages include `ops` and the `type` value is not a registered type, indicates a type that does not support the `publicKeyJwk` representation, the entire _Patch Action_ ****MUST**** be discarded, without any of it being used to modify the DID's state.

Reviewer: ```suggestion
    2. The object ****MUST**** include a type property, and its value ****SHOULD**** be the identifier type string of a registered [Cryptographic Suite](https://w3c-ccg.github.io/ld-cryptosuite-registry/) that supports JWK representations - for example:
```
